package.json

```
@@ -56,7 +56,7 @@
     "uppercamelcase": "^3.0.0"
   },
   "peerDependencies": {
-    "react": "^16.8.6 || ^17"
+    "react": "^16.8.6 || ^17 || ^18"
```

```suggestion
    "react": ">=16.8.6"
```

For the future, so we don't need to bump it with every React update and bother all the authors once again, specially considering this library only uses "main" React concepts